Update redoc.yaml

chatelao opened the pull request.

chatelao committed:
Update redoc.yaml

yves-bussard commented:
Willst Du den Trigger nicht auf **master** einschraenken? Pushen tust Du die erstellten Artefakte eh nur auf master und das macht konflikte.

Anstatt `on: [push]` -->

```
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

openbankingproject committed:
Add files via upload

yves-bussard reviewed (changes requested):
workflow anpassen (siehe Kommentar oben)

chatelao committed:
Merge branch 'master' into github-docs-for-bankable-assets

chatelao committed:
Update redoc.yaml

"master only" trigger integrieren

chatelao committed:
Update redoc.yaml

GitHub Pages:
- Build der Dokumentation vor und auf "master"
- Publikation nur auf "master" branch
